### DISCUSSION TITLE
Enable Build Artifacts in Actions?

### DISCUSSION THREAD

**Discussion by mapleexists** [FrozenBlock/WilderWild#135] [General]
@Soro335 @AViewFromTheTop @Treetrain1 this was disabled temporarily when the repo was private, but at this rate i don't entirely see why we can't add it back, unless we want to force people to pull the repository to play in-dev versions.. which, i guess is an understandable reason in that case. it does make it a little difficult for us to send wattles newer versions of the mod though.
BUT, we could probably just start making new releases for wattles

**Comment by EthanRStokes** [comment]
Just build the mod and send it to wattles

i don’t think actions will work anyways because frozenlib isn’t up to date

**Reply by mapleexists** [reply]
> Just build the mod and send it to wattles

every single time we've had to send him a version, i've been at work. the issue with that is that i use a chromebook and i can't enable dev mode, so i can't build it myself. and these days i literally can't even open twitter because the wifi is way too weak

> i don’t think actions will work anyways because frozenlib isn’t up to date

would there happen to be a way for it to pull frozenlib versions from the latest actions builds on the frozenlib repository? if we can figure that out then i can't see any reason why that wouldn't work. if anything that'd be super convenient for us

**Reply by EthanRStokes** [reply]
Maybe add a checkout frozenlib task in the actions script before it checks out wilder wild 
I’m not sure that would work because it would override files
it needs to get the dependency and store it somewhere else idk
